scripts/process-csv-images

```diff
@@ -30,7 +30,14 @@ RH_REGISTRY_OSBS=registry-proxy.engineering.redhat.com/rh-osbs
 DELOREAN_REGISTRY=quay.io/integreatly/delorean
 
 collect_csv_images() {
-    csv_images=(`grep -o $RH_REGISTRY_STAGE.* $1 || true`)
+    case $PRODUCT in
+    "fuse-online" | "3scale")
+        csv_images=(`grep -o $RH_REGISTRY_STAGE.* $1 || true`)
+        ;;
+    "amq-online")
+        csv_images=(`grep -o $RH_REGISTRY_PROD.* $1 || true`)
+        ;;
+    esac
```

Comment:
Rather than making this a per product thing (We have no idea what they might change this to), could we make it just collect all images that match any of these registries?

We will likely need to later down the line have some way of checking each of these images are actually available in the prod registry or not, but for now i guess we could assume they aren't.